app/views/appointments/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<h3>Your appointment</h3>
+
+<% if @appointments.empty? %>
+  <p class="text-general text-reading">
+    You don't have any appointments yet...
+  </p>
+<% else %>
+  <div class="table-responsive">
+    <table class="table table-striped table-text">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>First Name</th>
+          <th>Last Name</th>
+          <th>Email</th>
+          <th>Date of Appointment</th>
+          <th>Date of Create</th>
+          <th>Link</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @appointments.each do |appointment| %>
+          <tr>
+            <td><%= appointment.id %></td>
+            <td><%= appointment.first_name %></td>
```

Comment:
since appointment belongs_to user, you should get the first_name from the appointment's user, not from the appointment (unless you allow unregistered people to make appointments, but that's not the case)